Source/content_script.js

```diff
@@ -1,47 +1,43 @@
-walk(document.body);
-
-function walk(node) 
-{
-	// I stole this function from here:
-	// http://is.gd/mwZp7E
-	
-	var child, next;
-	
-	if (node.tagName.toLowerCase() == 'input' || node.tagName.toLowerCase() == 'textarea'
-	    || node.classList.indexOf('ace_editor') > -1) {
-		return;
-	}
-
-	switch ( node.nodeType )  
-	{
-		case 1:  // Element
-		case 9:  // Document
-		case 11: // Document fragment
-			child = node.firstChild;
-			while ( child ) 
-			{
-				next = child.nextSibling;
-				walk(child);
-				child = next;
-			}
-			break;
-
-		case 3: // Text node
-			handleText(node);
-			break;
-	}
-}
+walk(document.body)
+
+function walk (node) {
+  // I stole this function from here:
+  // http://is.gd/mwZp7E
+
+  let child, next
 
-function handleText(textNode) 
-{
-	var v = textNode.nodeValue;
+  if (!node) return
+  if (node.tagName === 'INPUT' || node.tagName === 'TEXTAREA') {
+    return
+  }
 
-	v = v.replace(/\bThe Cloud\b/g, "My Butt");
-	v = v.replace(/\bThe cloud\b/g, "My butt");
-	v = v.replace(/\bthe Cloud\b/g, "my Butt");
-	v = v.replace(/\bthe cloud\b/g, "my butt");
-	
-	textNode.nodeValue = v;
+  switch (node.nodeType) {
+    case 1: // Element
+    case 9: // Document
+    case 11: // Document fragment
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#Node_type_constants

you can use the nodeType constants

so 

```javascript
case Node.ELEMENT_NODE:
case Node.DOCUMENT_NODE:
case Node.DOCUMENT_FRAGMENT_NODE:
```

Reply:
La pli oni scias :thinking: :boom: